index.js

```diff
@@ -49,7 +50,12 @@ module.exports.transform = function(src, filename, options) {
   }
 
   if (filename.endsWith(".svg") || filename.endsWith(".svgx")) {
-    var jsCode = svgr.sync(src, { native: true });
+    var config = resolveConfig.sync(__dirname);
```

Comment:
One question, is this `__dirname` setting the folder where the configuration is? Because if so, wouldn't it point to your module's folder instead of the place where the user's app/`package.json` is?

Reply:
true, I think that I need to change it to something else, maybe `process.cwd()` or similar would work?